### PR TITLE
[wasm] Update build docs

### PR DESCRIPTION
**Pull request by kjpou1**

### CODE REVIEW

File: sdks/README.md [hunk 1]
```diff
@@ -31,49 +31,50 @@ it in your system's package manager](https://github.com/ninja-build/ninja/wiki/P
 Copy `Make.config.sample` to `Make.config` and edit the new file to disable building the target you're not interested in.
 Unless you have a very particular need, the BCL build should be left enabled as it's needed by all test runners.
 
-## Targets
+## Building for XA/XI/XM and WebAssembly
 
-To build Mono for Android, iOS or WebAssembly, the build scripts can be found in `sdks/builds/`. There are 5 predefined targets that can be invoked:
```
Comment (vargaz) [inline sdks/README.md]: Why is this removed ? Its just some general comments which still seem to be true.

Comment (kjpou1) [reply sdks/README.md]: Although, a comment about the `sdks/builds/out` should be mentioned.
